mongoCollections.ts

```diff
@@ -0,0 +1,18 @@
+import { dbConnection } from "./mongoConnection";
+
+// This will allow you to have one reference to each collection per app
+function getCollectionFn(collection: any) {
+	let _col: any = undefined;
+
+	return async () => {
+		if (!_col) {
+			const db = await dbConnection();
+			_col = await db.collection(collection);
+		}
+
+		return _col;
+	};
+}
+
+// List collections here in the mongoCollections object
+export const mongoCollections = { places: getCollectionFn("places") };
```

Comment:
What's the point of making an object of getCollection functions instead of an object of collection references themselves? Meaning, can we just return `db.collection(collection)` above instead of returning a function?

Reply:
I just feel like we can probably simplify this a bit, take a look and let me know if not.

Reply:
You're actually totallllllly right, we don't have to export an object of functions. I'll fix this!